### DISCUSSION TITLE
Screenshot Software on Mac?

### DISCUSSION THREAD

**Discussion by ItsMeBrianD** [diced/zipline#486] [Questions] [answered]
👋 Hey everybody

I've used Zipline for quite some time with Linux & Windows machines, using Flameshot (w/ helper script) on linux and ShareX on windows, but I can't find a good solution for automatically uploading screenshots on MacOS.

Does anybody have a good program / workflow for setting this up?

**Comment by diced** [comment, marked as answer]
you can use flameshot and the helper script (with some changes) on mac. I don't know of any other programs that are like mac-specific but flameshot works great on linux (from my experience), so I bet it will work great on mac.

The readme on this repo has some instructions for mac users too:
![image](https://github.com/diced/zipline/assets/35403473/bfb02211-6093-4a3b-8e45-b43bafc7bfe0)

this process will be simplified in v4 so you can automatically create a script that will be compatible with macos.

**Reply by ItsMeBrianD** [reply]
Oh I completely missed that; thank you!

Do you know if there is an easy way to set this as a keybind?

**Reply by diced** [reply]
I don't use macOS, but I think the app is called automate or something? Try searching up "how to run a shell script with a keybind on mac", or something similar.
